### DISCUSSION TITLE
How to even use Sentinel?

### DISCUSSION THREAD

**Discussion by theDanielJLewis** [coollabsio/coolify#4228] [Q&A] [answered]
I don't get it. I enabled Sentinel, but I don't know how to use it and there's nothing in the documentation or UI that helps. All I know is that it's enabled now.

The link from the release notes only points to the separate repo for Sentinel.

Is it a custom docker image?

**Comment by da411d** [comment]
Just randomly found out.
Honestly I expected more detailed statistics. Maybe also storage space etc.

![image](https://github.com/user-attachments/assets/5c17c370-e29f-4a9c-9ffd-294a65227d73)

![image](https://github.com/user-attachments/assets/cca152d8-2409-4377-9d99-7f886a6eb1e0)

**Reply by theDanielJLewis** [reply]
Oh. I was expecting some kind of central dashboard with the metrics.

It's also odd for them to be called "Sentinel" when that term isn't used anywhere else.

**Reply by da411d** [reply]
@theDanielJLewis totally agree with you

**Comment by mandeep511** [comment]
How do you enable Sentinel?

**Reply by LucaNerlich** [reply]
Settings (Server) -> General -> Sentinel Checkboxes -> View Metrics

**Comment by peaklabs-dev** [comment, marked as answer]
Sentinel docs are now live: https://coolify.io/docs/knowledge-base/server/sentinel/
